wpilibj/src/main/java/edu/wpi/first/wpilibj/TimedRobot.java

```diff
@@ -95,6 +96,24 @@ protected TimedRobot(double period) {
     HAL.report(tResourceType.kResourceType_Framework, tInstances.kFramework_Timed);
   }
 
+  /**
+   * Constructor for TimedRobot.
+   *
+   * @param period The period of the robot loop function.
```

Comment:
The doc for this overload should probably be consistent with the double overload and the C++ version.

FWIW I think this is _much_ clearer.